examples/pennylane_skip_plugins_hybrid.py

```diff
@@ -0,0 +1,391 @@
+r"""
+.. _plugins_hybrid:
+
+Plugins and Hybrid computation
+==============================
+
+This tutorial introduces the notion of hybrid computation by combining several PennyLane
+plugins. We first introduce PennyLane's `Strawberry Fields plugin <https://github.com/XanaduAI/pennylane-sf>`_
```

Comment:
Do we want to point to https://pennylane-sf.readthedocs.io/ ?

Reply:
I agree. In the tutorials, it might be nicer to refer to the docs. We already refer to the gihub page for all plugins here: https://pennylane.readthedocs.io/en/latest/plugins.html

Reply:
I think this is a good question. It would be nice to formalize around one link (github/rtd) as the main link we always use.

The only reason I tend to use github links is that I find the installation information is much more condensed and concise in the repo readme, and avoids the user getting bombarded by a huge new TOC to have to trawl through...

Reply:
I think I feel a more "professional" vibe from docs link than github link

Reply:
Oh interesting :laughing:, I'm the other way around!

Reply:
Fixed it to all point at readthedocs